### DISCUSSION TITLE
How can I merge two wallpapers in one /or/ how to get effects from Fluids and background from Matrix?

### DISCUSSION THREAD

**Discussion by Sat-Yo** [rocksdanister/lively#1834] [Q&A] [answered]
Hi! Help me please to explain how can I do something like this from Library wallpapers:

![Lively-report Matrix+Fluids](https://github.com/rocksdanister/lively/assets/141950737/56167c1a-6984-438a-bf56-70e102b15d50)

**Comment by rocksdanister** [comment]
Will require some coding, since both wallpapers are webgl canvases adding fluid with transparency ontop of matrix should work.

**Reply by Sat-Yo** [reply]
Can you please briefly tell me how to do this exactly? I can code a little

**Reply by rocksdanister** [reply, marked as answer]
Combine into single webpage, add fluid canvas ontop of matrix one with transparency.
https://github.com/rocksdanister/WebGL-Fluid-Simulation/blob/453d97445ca186d1d8195d2d8dfdc4a9bc272bce/fluid.html#L104

https://github.com/bad1dea/lively_matrix/blob/4abb87ed72ecf88d48da4543098489a2cc740800/index.html#L9

**Reply by Sat-Yo** [reply]
Thank you very much!

**Reply by Sat-Yo** [reply]
I can't understand how to add transparency. Can you help, please?

**Reply by rocksdanister** [reply]
https://github.com/rocksdanister/WebGL-Fluid-Simulation/blob/453d97445ca186d1d8195d2d8dfdc4a9bc272bce/js/script.js#L46

**Reply by Sat-Yo** [reply]
Meh, sorry, I don't understand how to do it :(
I switched transparent to true, but it looks like I'm doing other things wrong.
Should I just copy lines 104-105 from Fluids and paste it ontop of line 9 in Matrix?

**Reply by Sat-Yo** [reply]
I'm really sorry * shame *

**Reply by Sat-Yo** [reply]
Maybe you can give some tutorial link? Don't really want to bother you

**Reply by rocksdanister** [reply]
Can't say for sure the problem without checking it out myself but I don't have time right now for that.

Fluid already has a transparency feature and that is how the background image is shown, in theory showing the matrix canvas instead should also work.

**Reply by kenwud** [reply]
id love to see the results from this if you manage to do it. or if  u need some help or anything i could try to help figure out how to help do it

is there a lively discord or anything yet?
